src/dev/build/tasks/os_packages/create_os_package_tasks.ts

```diff
@@ -60,12 +60,21 @@ export const CreateRpmPackage: Task = {
   },
 };
 
-export const CreateDockerPackage: Task = {
-  description: 'Creating docker package',
+export const CreateDockerAMD64Package: Task = {
+  description: 'Creating docker amd64 package',
 
   async run(config, log, build) {
     // Builds Docker targets for default and oss
-    await runDockerGenerator(config, log, build);
+    await runDockerGenerator(config, log, build, false, 'amd64');
```

Comment:
Can we use `x64` and `aarch64` to maintain consistency with Node the the build?

https://github.com/elastic/kibana/blob/master/src/dev/build/lib/platform.ts#L59-L64

Reply:
Added with 4cccd58 and the few followup commits for fixes.